quota/crdbqm/common_test.go

```diff
@@ -37,7 +37,10 @@ func TestMain(m *testing.M) {
 	dburl.Path = "/"
 
 	// Set the environment variable for the test server
-	os.Setenv(testdb.CockroachDBURIEnv, dburl.String())
+	if err := os.Setenv(testdb.CockroachDBURIEnv, dburl.String()); err != nil {
+		klog.Errorf("Failed to set CockroachDBURIEnv: %v", err)
+		os.Exit(1)
```

Comment:
If you use `klog.Exitf` you'll get an `os.Exit(1)` thrown in for free :)

Reply:
I've done it this way for consistency with L29-30. For better or worse.